modules/obj/README.md

```diff
@@ -3,3 +3,5 @@
 [loaders.gl](https://uber-web.github.io/loaders.gl/#/docs) is a collection of framework independent 3D and geospatial parsers and encoders.
 
 This module contains loader for the OBJ format.
+
+Sample OBJ files are from John Burkardt's [data sample collection](https://people.sc.fsu.edu/~jburkardt/data/data.html)
```

Comment:
This is a strange place to put this info. 

We usually add such info to a README.md to `test/data` in each module folder. We also mention what license those files are copied under.